index.html

```diff
@@ -24,6 +26,8 @@
     <div data-note="B" class="key white"></div>
   </div>
 
+  
```

Comment:
Hei, bro!
I think you should remain comments that will help user can read code more easily
Thank you!